NitroxModel/Packets/CyclopsChangeColor.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using UnityEngine;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class CyclopsChangeColor : AuthenticatedPacket
+    {
+        public String Guid { get; private set; }
```

Comment:
minor: just a note for the future, we stopped doing private set in anemic data objects